app/page.jsx

```diff
@@ -1,5 +1,22 @@
+import Image from "next/image";
+import SearchBox from "./components/SearchBox";
 import styles from "./page.module.css";
 
 export default function Home() {
-  return <main className={styles.main}></main>;
+  return (
+    <main className={styles.main}>
+      <SearchBox />
+      {/* this image can be changed pretty easily, however it seems like this
+      format for loading images is the most optimized for Next, better than
+      setting it in the CSS */}
+      <Image
+        className={styles.image}
+        src="/AT_CD_04.png"
+        loading="lazy"
```

Comment:
don't actually lazy load this image, it's the largest contentful paint and lazy loading it will delay that metric